src/Search/Workflow/BlogArticleWorkflow.php

```diff
@@ -81,11 +81,15 @@ public function validate(BlogArticle $blogArticle) : BlogArticle
      */
     public function index(BlogArticle $blogArticle) : array
     {
-        // This step is still not used very much.
         $this->logger->debug('BlogArticle<'.$blogArticle->getId().'> Indexing '.$blogArticle->getTitle());
+        // Normalized fields.
+        $blogArticleObject = json_decode($this->serialize($blogArticle));
+        $sortDate = $blogArticle->getPublishedDate();
+        $blogArticleObject->sortDate = $sortDate->format('Y-m-d\TH:i:s\Z');
```

Comment:
this assignment to sortData should be extracted, the format for example is a detail that should be kept in one place. Method in a trait or abstract class?